legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -59,8 +59,11 @@ def get_businesses(identifier: str):
 
     # getting all business info is expensive so returning the slim version is desirable for some flows
     # - (i.e. business search update)
+    # need to add the alternateNames array here because it is not a part of slim JSON
     if str(request.args.get('slim', None)).lower() == 'true':
-        return jsonify(business=business.json(slim=True))
+        business_json = business.json(slim=True)
+        business_json['alternateNames'] = business.get_alternate_names()
```

Comment:
@kialj876 @argush3 @seeker25 Are you guys OK with this being here? This'll keep the flow where we grab the alternateNames when calling the business endpoint. This'll keep the flow working on Kial's side as usual:
![slim true call](https://github.com/bcgov/lear/assets/122301442/d8a91870-aca3-45d0-9197-5949f12cc7f7)

Reply:
you moved the alternateNames out of the .json(slim=True)? yup as long as it's included in /search I think it is?

Reply:
Yep, exactly. It's also still included in /search as long as it's a firm (this is what MBR uses).